Main Menu with VR selection

stedixon opened the pull request:
Adds VR selection to the main menu.

The main menu should appear separate from the maze generation, unlike during the previous demo.

stedixon committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into MainMenu

stedixon committed:
Enable VR Selection

Scene is now VR enabled and VR selection works for all buttons.
Future Update Required: set time before 'click' occurs.

stedixon committed:
VR selection Update

Buttons now change color when being looked at by the user. User is able
to select via VR selection.

JStuve reviewed (approved):
Good to go, working on movement!

stedixon committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into MainMenu

stedixon committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into MainMenu

stedixon committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into MainMenu

# Conflicts:
#	VR Maze/Assets/Editor.meta
#	VR Maze/Assets/Resources.meta

stedixon committed:
Revert "Merge remote-tracking branch 'refs/remotes/origin/master' into MainMenu"

This reverts commit a179e8d9722b8d4ca71456a1d8ae8c2c2835bc9a.

stedixon committed:
synced with master

Synced to master branch to test interaction with current scenes and
scripts.

All items work. For some reason my AutoMaze generate scene lost it's
walking script during sync with master so I re-added the walking script
and all necessary components.
